### PR TITLE
PLANET-3754 - Convert Timeline Block to Gutenberg

### PR DESCRIPTION
**Pull request by comzeradd**

### CODE REVIEW

File: react-blocks/src/blocks/Timeline/Timeline.js [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+        <div>
+          <h2>{__('Timeline options', 'p4ge')}</h2>
+          <p><i>{__(
+            'Display a timeline from a Google Sheet',
+            'p4ge'
+          )}</i></p>
+
+          <div>
+            <TextControl
+              label="Timeline Title"
```
Comment (sagarsdeshmukh) [inline react-blocks/src/blocks/Timeline/Timeline.js]: Need to use translator function for labels and placeholder text.

File: react-blocks/src/blocks/Timeline/TimelineBlock.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
+import {Timeline} from './Timeline.js';
+
+export class TimelineBlock {
+  constructor() {
+    const {registerBlockType} = wp.blocks;
+
+    registerBlockType('planet4-blocks/timeline', {
+      title: 'Timeline',
```
Comment (sagarsdeshmukh) [inline react-blocks/src/blocks/Timeline/TimelineBlock.js]: Please use string translation function(`__()`) here.

**Comment by comzeradd** [comment]
@sagarsdeshmukh thanks! PR updated.

**Comment by sagarsdeshmukh** [comment]
Hi @comzeradd - While testing Timeline block on local, I am facing below error -
image

**Comment by pablocubico** [comment]
Hey @comzeradd I just found a few HTML tags unparsed in this help text:

image

Also, the timeline is rendering on the frontend, but it's super tall, I'm getting this weird super height number: 

image

**Comment by comzeradd** [comment]
@sagarsdeshmukh PR updated. I added a `componentDidMount` method so the Timeline also works on first load in the backend.

@pablocubico I'm not able to reproduce that. After adding `RawHTML` the help text is rendered as html in the backend. On the frontend it also looks good, but I think we haven't still fix the css loading issue, so if you disabled plugin-blocks you don't get proper css.

Screenshot from 2019-08-21 09-06-42

**Comment by pablocubico** [comment]
@comzeradd Right then, I merged https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/17 into your branch locally just to load the frontend styles and the height issue went away, now I'm getting two different arrows on the sides, but this is happening only on the frontend, so I guess we'll need to keep an eye on that when merging the frontend styles PR. I think this one can be merged.


image

**Comment by comzeradd** [comment]
I'll merge it to see it live and I'll leave a comment to our progress document to keep an eye for this issue.

**Pull request merged**